overlay/ports/qt5-base/portfile.cmake

```diff
@@ -163,14 +163,14 @@ elseif(VCPKG_TARGET_IS_OSX)
         string(REGEX MATCH "^[0-9][0-9]\.[0-9][0-9]*" VCPKG_OSX_DEPLOYMENT_TARGET ${VCPKG_OSX_DEPLOYMENT_TARGET})
         message(STATUS "Major.Minor OSX SDK Version: ${VCPKG_OSX_DEPLOYMENT_TARGET}")
         set(ENV{QMAKE_MACOSX_DEPLOYMENT_TARGET} ${VCPKG_OSX_DEPLOYMENT_TARGET})
-        if(${VCPKG_OSX_DEPLOYMENT_TARGET} GREATER "10.14") # Max Version supported by QT
-            message(STATUS "Qt 5.12.4 only support OSX_DEPLOYMENT_TARGET up to 10.14")
-            set(VCPKG_OSX_DEPLOYMENT_TARGET "10.14")
+        if(${VCPKG_OSX_DEPLOYMENT_TARGET} GREATER "10.15") # Max Version supported by QT. This version is defined in mkspecs/common/macx.conf as QT_MAC_SDK_VERSION_MAX
```

Comment:
Why is this change needed if we use 10.12 as the deployment target?

Reply:
This is bugfix from upstream.

Reply:
I'm unclear where it comes from. Is it backported from a later Qt version upstream?